packages/lesswrong/components/users/NewUserCompleteProfile.tsx

```diff
@@ -28,6 +35,10 @@ const styles = (theme: ThemeType): JssStyles => ({
       color: theme.palette.grey[600],
     },
   },
+  sectionHeadingText: isEAForum ? {
```

Comment:
I don't think less wrong uses this page at all? So shouldn't be necessary — just make it good for anyone using it, IMO.

Reply:
Oh interesting, I didn't know that, let me check with them